### DISCUSSION TITLE
PDF preview switching back to even spreads setting when file updates

### DISCUSSION THREAD

**Discussion by julpen** [James-Yu/LaTeX-Workshop#4113] [Q&A]
My setting for "Spread Mode" is 0, which works fine when i open my PDF file for preview, resulting in single-page view.

But I now run into the problem that anytime I regenerate the PDF with the preview open, it sets the preview window to "Even Spreads", messing up the view. I assume that it is related to the document that I am working on which does use a double page layout, but I have never had the problem before, having used the extension for a while already.

Is there some setting that I am missing or was there a new feature introduced that might cause this issue?

**Comment by James-Yu** [comment]
#4104

**Comment by julpen** [comment]
Well now I feel silly, no idea why I didn't find that issue. Is there an ETA for the next release that will actually include the fix?
